cmd/gocritic/testdata/docStub/positive_tests.go

```diff
@@ -1,7 +1,19 @@
 package checker_test
 
+type kek struct{}
+
 // Foo ...
 /// silencing go lint doc-comment warnings is unadvised
-func Foo() {
+func Foo() {}
+
+// Wow
+/// silencing go lint doc-comment warnings is unadvised
+func (k *kek) Wow() {}
 
-}
+//Well
+/// silencing go lint doc-comment warnings is unadvised
+func Well() {}
+
+// Goo go go.
+/// silencing go lint doc-comment warnings is unadvised
+func Go() {}
```

Comment:
This doesn't look like something `docStub` should report as "silencing go lint".
It's also not a valid doc comment and go lint will trigger on it.
`docStub` aims to find comments that are obviously "hacks" to avoid go lint complaints.
Like:
```go
// Foo ...
func Foo() {}
```
`go lint` would not trigger on this, but `docStub` should. It's a tool that makes it easier to refactor such code that used doc comment stubs.